tests/test_registration_manager.py

```diff
@@ -0,0 +1,132 @@
+from abc import ABCMeta
+
+from six import with_metaclass
+from zipline.extensions import Registry
+from zipline.testing.fixtures import ZiplineTestCase
+from zipline.testing.predicates import (
+    assert_equal,
+    assert_is,
+    assert_raises_str,
+    assert_true,
+)
+
+
+class FakeInterface(with_metaclass(ABCMeta)):
```

Comment:
Does this actually need to be an abc?

Reply:
It doesn't need to be, but it does make it more realistic if we have it this way (the registry would usually be instantiated with an abc)

Reply:
This makes me think that it is a requirement, even if it is not. For unit tests, it is best to keep the test limited to just a small component under test.

Reply:
Ok, makes sense